src/v2/guide/forms.md

```diff
@@ -4,20 +4,20 @@ type: guide
 order: 10
 ---
 
-## Basic Usage
+## Penggunaan Dasar
 
-You can use the `v-model` directive to create two-way data bindings on form input, textarea, and select elements. It automatically picks the correct way to update the element based on the input type. Although a bit magical, `v-model` is essentially syntax sugar for updating data on user input events, plus special care for some edge cases.
+Anda bisa menggunakan `v-model` untuk membuat binding data dua arah di *form input*, *textarea*, dan *select elements*. Hal tersebut secara otomatis akan memilih cara yang tepat untuk memperbarui elemen berdasarkan tipe *input*. Meskipun agak ajaib, `v-model` pada dasarnya adalah *syntax sugar* untuk memperbarui data setiap ada aksi *input* oleh pengguna, Ditambah perawatan khsusus untuk beberapa *edge cases*.
 
-<p class="tip">`v-model` will ignore the initial `value`, `checked` or `selected` attributes found on any form elements. It will always treat the Vue instance data as the source of truth. You should declare the initial value on the JavaScript side, inside the `data` option of your component.</p>
+<p class="tip">`v-model` akan mengabaikan atribut `value`, `checked` atau `selected` awal yang ditemukan pada elemen *form* apapun. Itu akan selalu memperlakukan data *Vue Istance* sebagai sumber kebenaran. Anda harus mendeklarasikan nilai awal pada sisi *Javascript*, Di dalam opsi `data` dari komponen anda.</p>
```

Comment:
- apapun lebih baik ganti jadi manapun.
- "Itu akan selalu memperlakukan" lebih cocok pakai "Hal tersebut akan selalu memperlakukan" agar lebih natural
- sumber kebenaran agak aneh sih, tetap jadi source of truth dengan italic saja.
- anda di akhir, ganti jadi Anda.